scripts/eda_dev.sh

```diff
@@ -81,10 +82,17 @@ check_uvicorn_status() {
 
 add-dev-user() {
   log-info "Adding dev user: ${DEV_USER}, pass: ${DEV_PASS}"
-  log-debug "scripts/adduser.py --password ${DEV_PASS} ${DEV_USER}"
+  log-debug "${DEV_SCRIPTS_PATH}/adduser.py --password ${DEV_PASS} ${DEV_USER}"
   "${DEV_SCRIPTS_PATH}"/adduser.py --password "${DEV_PASS}" "${DEV_USER}"
```

Comment:
Maybe not related to this PR, but I think we should include the superuser (-S) flag here?

Reply:
I fixed the script and removed the dev:user:add task